src/common/state/ColorTableAttributes.code

```diff
@@ -114,16 +114,17 @@ Definition:
 // Creation:   Sat Jun 16 20:32:23 PST 2001
 //
 // Modifications:
+//   Changed ColorTableAttributes `names` to `colorTableNames`.
```

Comment:
Add name and date to comment and remove extra blank line below the change comment.